packages/twenty-server/src/engine/core-modules/billing/utils/billing-webhook-subscription-transformer.util.ts

```diff
@@ -0,0 +1,79 @@
+import Stripe from 'stripe';
+
+import { BillingSubscriptionCollectionMethod } from 'src/engine/core-modules/billing/enums/billing-subscription-collection-method.enum';
+import { SubscriptionStatus } from 'src/engine/core-modules/billing/enums/billing-subscription-status.enum';
+
+export const transformStripeSubscriptionEventToSubscriptionRepositoryData = (
+  workspaceId: string,
+  data:
+    | Stripe.CustomerSubscriptionUpdatedEvent.Data
+    | Stripe.CustomerSubscriptionCreatedEvent.Data
+    | Stripe.CustomerSubscriptionDeletedEvent.Data,
+) => {
+  return {
+    workspaceId,
+    stripeCustomerId: data.object.customer as string,
```

Comment:
as string?

Reply:
Because
data.object.customer is of type
(property) Stripe.Price.product: string | Stripe.Product | Stripe.DeletedProduct

This is the same method we use to upsert data in BillingSubscriptionItemRepository (it hasn't changed). Do we need to refactorize?